llvm/lib/IR/Verifier.cpp

```diff
@@ -137,6 +137,10 @@ static cl::opt<bool> VerifyNoAliasScopeDomination(
     cl::desc("Ensure that llvm.experimental.noalias.scope.decl for identical "
              "scopes are not dominating"));
 
+static cl::opt<bool> AllowAtomicVector(
```

Comment:
This patch should not touch the verifier, and there should never be an option to make IR valid or not